src/components/inputs/CommentInput.tsx

```diff
@@ -26,15 +26,9 @@ export const CommentInput = ({ createComment, task_id }: Prop) => {
   const currentUserId = tokenPayload?.clientId ?? tokenPayload?.internalUserId
   const currentUserDetails = assignee.find((el) => el.id === currentUserId)
 
-  const handleSubmit = (opts: { isEnter: boolean } = { isEnter: false }) => {
-    const detailsText = detail.replaceAll('<p>', '').replaceAll('</p>', '').replaceAll(' ', '').replaceAll('<br>', '')
-    const isEmpty = detailsText === ''
-    if (isEmpty) return
-
-    // Tiptap / Hardbreak appends this at the end if you submit a comment using enter
-    const enterBreakEl = '<p></p>'
+  const handleSubmit = () => {
     const commentPayload: CreateComment = {
-      content: opts?.isEnter ? detail.slice(0, detail.length - enterBreakEl.length) : detail,
+      content: detail,
```

Comment:
@aatbip Tiptap sends an extra `<p></p>` at the end when submitting a comment using enter key. Where are we handling this?

Reply:
I don't see a version bump for `tippytappy` either... where is this being handled instead?

Reply:
This is working because we made some changes in the `HardBreak` extension's configuration in `tippytappy` yesterday. Regarding extra `p` tags at the end, seems like this has also been resolved and not happening anymore. Since the `enter` command line break is already disabled, `p` tag won't come at all. Please see the video below - 
[Screencast from 25-10-24 01:53:45 PM +0545.webm](https://github.com/user-attachments/assets/40fd125f-698f-4b11-ba5d-1106ee994f1b)